sourcegraph/about
# Enhance mobile exp

Enhancement to mobile experience.

### Changelog
- Navbar width fits screen to match the body
- Navbar logo height matches the hamburger menu height
- Shows one action at a time: Cookie Banner covers Chat Bot until banner is complete (on mobile & desktop)
- Adjusts vertical padding on homepage for mobile (Vert padding between sections should be 4rem (py-5) on mobile and 6rem otherwise (py-7)) ([Figma](https://www.figma.com/file/o1QRtdQI0ozKq0n7ATrKlx/Marketing-DLS?node-id=7649%3A36393))

### Notes
- No change for Acceptance Criteria #3 because I couldn't repro a large cookie banner. Showing one banner/bot at a time solves the noise of this UI bug.
- I've updated #5381 to note the spacing standardizations so we can apply these to every page at some point

### Test
1. Ensure prettier has standardized the proposed changes.
2. Ensure above UI script bugs are cleared on mobile & web
3. Ensure mobile navbar is compliant to design standards (including dark navbar: i.e. the _Cern Case Study_)


**BEFORE**:
![Screen Shot 2022-06-17 at 12 35 17 PM](https://user-images.githubusercontent.com/59381432/174368990-f755aecd-0377-4d6f-b7ab-e2f99df40941.png)
![Screen Shot 2022-06-17 at 12 35 27 PM](https://user-images.githubusercontent.com/59381432/174369366-14bbb13a-f750-458b-8a98-493897d2426b.png)

**NOW**:
![Screen Shot 2022-06-17 at 12 32 57 PM](https://user-images.githubusercontent.com/59381432/174369624-4d769676-a35a-4efe-9856-307d9ee9957a.png)
![Screen Shot 2022-06-17 at 12 33 08 PM](https://user-images.githubusercontent.com/59381432/174369785-e1838e70-416f-462b-be7d-5ea25b61534e.png)
